unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/GotoPanelHUD/GotoPanelHUDController.cs

```diff
@@ -0,0 +1,45 @@
+using DCL;
+using UnityEngine;
+
+public class GotoPanelHUDController : IHUD
+{
+
+    internal GotoPanelHUDView view { get; private set; }
+
+    public GotoPanelHUDController()
+    {
+        view = UnityEngine.Object.Instantiate(Resources.Load<GameObject>("GotoPanelHUD")).GetComponent<GotoPanelHUDView>();
```

Comment:
This is not mockable I would do the following:

1) Extract a `IGoToPanelHUDView` interface.
2) `GoToPanelHUDController` would only interact with `IGoToPanelHUDView`.
3) Have an `internal virtual IGoToPanelHUDView CreateView()` method that would return the real view so it can be overridden in tests.

I would also have a static method `GoToPanelHUDView.CreateView` with all the nuisances of resources and the instantiation, so we can call that in our virtual method `CreateView`.

For further info take a look at huds such as SignUpHUD.